persistence/ravendb/cluster-configuration_config_raven_ [6,7].partial.md

```diff
@@ -0,0 +1,3 @@
+If the database is configured as a multi-node cluster, remove all nodes but one.
```

Comment:
it looks like there is a space in the file name after the last `_`?

Reply:
The fix existed (we didn't create it), but I'll check it out.